app/views/todo_tasks/index.html.erb

```diff
@@ -4,13 +4,18 @@
 
 <div id="todo_tasks">
   <% @todo_tasks.each do |todo_task| %>
+    <% if todo_task.isCurrent %>
```

Comment:
Only show tasks for current user

Q: will this query all tasks and then iterate? Would it be more performant to query tasks based on username and print?